ssl/t1_lib.c

```diff
@@ -1499,6 +1741,13 @@ int tls12_check_peer_sigalg(SSL_CONNECTION *s, uint16_t sig, EVP_PKEY *pkey)
             pkeyid = EVP_PKEY_RSA_PSS;
     }
     lu = tls1_lookup_sigalg(s, sig);
+    /* if this sigalg is loaded, set so far unknown pkeyid to its sig NID */
+    if ((pkeyid == -1) && (lu != NULL)) pkeyid = lu->sig;
```

Comment:
nit: put `pkeyid = lu->sig` on the next line

Reply:
So changed in 0ac814f8247d1a5f157eec2b62bce0000c3a2d1e.